index.html

```diff
@@ -4,52 +4,42 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./style.css">
-    <link href="https://font.googleapis.com/icon?family=Material+Icons"
-        rel="stylesheet">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    
 </head>
 <body>
-    <header>
-        <nav> id="nav">
-            <a class="logo" href=""#>My Logo</a>
+       <header>
+        <nav class="nav-bar">
+            <a href="#">My Logo</a>
 
-            <nav><img src="/image/Union.png" alt="Menu"></nav>
-        </nav>
-    </header>
-    <section>
-        <h1>I'am Devesh<br>Glad to see<br> you!</h1>
-        <p> I'am a software developer! I can help you<br> build a product ,
-            feature or website Look<br> through some of my work and<br> experience!
-            If you like what you see and<br> have a project you need coded,<br>
-            don't hestiate to contact me.
-        </p>
-        <footer>
-            <h3>LET'S CONNECT</h3>
-            <div>
-                <ul>
-                    <li class=""icon><a href="#"><img
-                        src="/images/twitter.png"
-                        alt="twitter link"></a></li>
-                    <li class="icon"><a href="#"><img 
-                        src="/images/Linkedin_icon.png"
-                         alt="linked link"></a></li>
-                    <li class="icon"><a href="#"><img
-                         src="/images/M.png" alt="M"></a>
-                    </li>
-                    <li class="icon"><a href="#"><img 
-                        src="/images/github.png" alt="github
-                        link"></a></li>
-                    <li class="icon"><a href="#"><img 
-                        src="/images/doits.png" alt="doit"></a></li>    
-                </ul>
-            </div>
-        </footer>
-        </section>
-        </body>
-</html>
+               <img src="image/Union.png" alt="Menu">
+              </nav>
+                           
+        </header>
+        <section class="main-section">
+               <h1 class="heading">I'am Devesh<br>Glad to see you!</h1>
+                      <p class="introduction-1">I’m a software developer! I can help you <br>build a product, feature or website Look <br>through some of my work and <br>experience! If you like what you see and <br>have a project you need coded, don’t<br> hestiate to contact me.</p>
+                      <h3 class="introduction-2">LET'S CONNECT</h3>
+                      <ul class="nav-link">
+                        <li>
+                            <a href="#"><img src="image/Twiter.png" alt="twitter"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="image/Li.png" alt="linkedin"></a>
+                            <a href="#"><img src="image/Ln.png" alt="linkedin"></a> 
+                        </li>
+                        <li>
```

Comment:
- [ ] Great job @Devesh123987 Your Project looks great. There is just a slight problem. I noticed that you have a vertical scroll in your project and this can be a frustrating user experience. So I will suggest that you fix it to improve the overall user experience in your proje:+1: 
- [ ] I have attached a video below for your reference :+1: 
[Screencast from 03-25-2023 12:00:57 PM.webm](https://user-images.githubusercontent.com/102699958/227717533-7df31237-142f-4cc4-879d-bbc283504f64.webm)